gilbreth_support/urdf/break_beam/break_beam.xacro

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0"?>
 <robot name="break_beam" xmlns:xacro="http://www.ros.org/wiki/xacro">
-  <xacro:macro name="break_beam" params="prefix">
+  <arg name="prefix" default="break_beam_"/>
 
-    <!-- URDF -->
-    <xacro:include filename="$(find gilbreth_support)/urdf/break_beam/break_beam.urdf.xacro"/>
-    <xacro:break_beam_urdf prefix="${prefix}"/>
+  <!-- URDF -->
+  <xacro:include filename="$(find gilbreth_support)/urdf/break_beam/break_beam.urdf.xacro"/>
+  <xacro:break_beam_urdf prefix="$(arg prefix)"/>
 
-    <!-- Gazebo -->
-    <xacro:include filename="$(find gilbreth_support)/urdf/break_beam/break_beam.gazebo.xacro"/>
-    <xacro:break_beam_gazebo prefix="${prefix}"/>
+  <!-- Gazebo -->
+  <xacro:include filename="$(find gilbreth_support)/urdf/break_beam/break_beam.gazebo.xacro"/>
+  <xacro:break_beam_gazebo prefix="$(arg prefix)"/>
 
-  </xacro:macro>
 </robot>
```

Comment:
What's the difference between this and the file above?

Reply:
Can you clarify which file above you are referring to?